packages/@o3r/telemetry/src/environment/index.ts

```diff
@@ -78,15 +110,15 @@ export const getEnvironmentInfo = () => {
   const otterInfo = {
     version: otterCorePackageJsonPath ? JSON.parse(fs.readFileSync(otterCorePackageJsonPath, { encoding: 'utf-8' })).version as string : undefined
   };
-  return { os: osInfo, node: nodeInfo, packageManager: packageManagerInfo, otter: otterInfo };
+  return { os: osInfo, node: nodeInfo, packageManager: packageManagerInfo, otter: otterInfo, ci: process.env.CI === 'true' };
```

Comment:
we could use the same check as angular for that https://github.com/angular/angular-cli/blob/4cbfe5fe01c1c98c9eac63d38fc62e3c1ee80c0b/packages/angular_devkit/schematics_cli/bin/schematics.ts#L480